### PR TITLE
Add showcase tag catalyst and showcase Lido Nation

### PR DESCRIPTION
**Pull request by fill-the-fill**
* Name: Lido Nation
* Description: Series of experiments which seeks to generate the highest levels of community innovation.
* Link:   [lidonation](https://www.lidonation.com/en/project-catalyst/projects)
* Tags:
  * `catalyst`
  * `analytics`

### CODE REVIEW

File: src/data/showcases.js [hunk 1]
```diff
@@ -190,6 +190,14 @@ export const Tags = {
     color: '#800080'
   },
 
+   // Catalyst
+   catalyst: {
+    label: "Catalyst",
+    description:
+      "Series of experiments which seeks to generate the highest levels of community innovation",
```
Comment (rdlrt) [inline src/data/showcases.js:197]: Not sure if intentional, but seems same description as project itself
``` suggestion
Projects that aid to Cardano's interim governance programme (Catalyst)
```

File: src/data/showcases.js [hunk 2]
```diff
@@ -1015,6 +1023,15 @@ const Showcases = [
     source: null,
     tags: [ "pooltool", "analytics"],
   },
+  {
+    title: "Lido Nation",
+    description:
+      "Series of experiments which seeks to generate the highest levels of community innovation.",
```
Comment (rdlrt) [inline src/data/showcases.js:1029]: I see it more as a aggregation of of existing catalyst proposals, results, feedbacks, entities, etc (as against any experiments that were innovated themselves), but do not feel too strongly about the description

Comment (fill-the-fill) [reply src/data/showcases.js:1029]: How about 'Series of existing catalyst proposals, results, feedbacks and many more'

What do you think?

Comment (rdlrt) [reply src/data/showcases.js:1028]: Ye - sounds better (series or aggregation - either)

**Review by rdlrt — APPROVED** [review]

**Review by os11k — APPROVED** [review]

**Review by katomm — APPROVED** [review]
👍

**Pull request merged**